cor_tabular/code/make_table_all.R

```diff
@@ -410,16 +419,18 @@ tab_dm <- full_join(
                                            "Communities of Color",
                                            "Hispanic or Latino","Not Hispanic or Latino",
                                            "Not reported and unknown ",
-                                           "At-risk","Not at-risk"))) %>% 
+                                           "At-risk","Not at-risk",
+                                           "Age $<$ 65 At-risk","Age $<$ 65 Not at-risk",
```

Comment:
Why not $<$ as you do for `$\\geq$`?

Reply:
Sorry not sure what the question means...

Reply:
Sorry for both questions, I am asking why you are not using Latex math expression `$<$` like you do for the greater than or equal to sign `$\\ge$`